src/timetable_gui.cpp

```diff
@@ -561,11 +561,29 @@ struct TimetableWindow : Window {
 	{
 		const Vehicle *v = this->vehicle;
 		Rect tr = r.Shrink(WidgetDimensions::scaled.framerect);
+		TimerGameTick::Ticks preci;
```

Comment:
I don't think we need to worry about saving four characters from `precision`

Reply:
Thank you for your review, LordAro. I have a quick question. Is it a good habit to initialize 'precision' as 'TimerGameTick::Ticks' here or would 'uint' be totally fine?